### PR TITLE
Updated to akka 2.6.3, akka http to 10.1.11, akka-stream-contrib to 0…

### PR DESCRIPTION
**Pull request by RayRoestenburg**
### What changes were proposed in this pull request?

Update Akka version to latest

### Why are the changes needed?

Keep up with latest version, cleanup build warnings.

### Does this PR introduce any user-facing change?

Users can use Akka 2.6.x features from  this point in AkkaStreamlet.

### How was this patch tested?

All tests green. CI green.
All examples projects build and tests green, runLocal no issues.

### CODE REVIEW

File: core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala [hunk 1]
```diff
@@ -96,13 +95,6 @@ object Main extends {
 
   private def exitWithFailure() = System.exit(-1)
 
-  private def createMaterializer()(implicit system: ActorSystem) = {
-    val decider: Supervision.Decider = {
-      case _ ⇒ Supervision.Stop
-    }
-    ActorMaterializer(ActorMaterializerSettings(system).withSupervisionStrategy(decider))
-  }
-
```
Comment (agemooij) [inline core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala]: @RayRoestenburg does this change the behavior when a stream crashes? Or is `Stop` the default now?

Comment (agemooij) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala]: ah, forget it, I just noticed it got moved to custom attributes 😉

File: core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala [hunk 2]
```diff
@@ -106,8 +106,9 @@ trait SparkStreamlet extends Streamlet[SparkStreamletContext] with Serializable
           // stop all query execution
           streamletQueryExecution.stop()
 
-        } else ()
+        }
       }
+      val done = system.scheduler.scheduleWithFixedDelay(InitialDelay, MonitorFrequency)(runnable)
```
Comment (agemooij) [inline core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala]: Just curious why you made this change. Readability?

File: core/project/Dependencies.scala [hunk 3]
```diff
@@ -13,48 +13,48 @@ object Version {
...
-  val Config                = "com.typesafe"           % "config"                   % "1.3.4"
-  val Logback               = "ch.qos.logback"         % "logback-classic"          % "1.2.3"
+  val AkkaHttp               = "com.typesafe.akka"     %% "akka-http"                 % Version.AkkaHttp
+  val AkkaHttpJackson        = "com.typesafe.akka"     %% "akka-http-jackson"         % Version.AkkaHttp
+  val AkkaHttpSprayJson      = "com.typesafe.akka"     %% "akka-http-spray-json"      % Version.AkkaHttp
+  val AkkaStream             = "com.typesafe.akka"     %% "akka-stream"               % Version.Akka
+  val AkkaSlf4j              = "com.typesafe.akka"     %% "akka-slf4j"                % Version.Akka
+  val AkkaStreamContrib      = "com.typesafe.akka"     %% "akka-stream-contrib"       % "0.11"
+  val AkkaStreamKafka        = "com.typesafe.akka"     %% "akka-stream-kafka"         % Version.AlpakkaKafka exclude("com.fasterxml.jackson.core","jackson-databind") exclude("com.fasterxml.jackson.module", "jackson-module-scala")
+  val AkkaStreamKafkaTestkit = "com.typesafe.akka"    %% "akka-stream-kafka-testkit" % Version.AlpakkaKafka intransitive()
```
Comment (agemooij) [inline core/project/Dependencies.scala]: OMG, a misaligned line!!!

File: core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala (hunk 2 above)
Comment (RayRoestenburg) [reply core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala]: deprecation

**Pull request merged**